plugins/modules/synchronize.py

```diff
@@ -619,14 +632,17 @@ def _write_password_to_pipe(proc):
     out_lines = out_clean.split('\n')
     while '' in out_lines:
         out_lines.remove('')
+
+    result = dict(changed=changed, rc=rc, cmd=cmdstr, stdout_lines=out_lines, msg=out_clean)
+
+    if quiet:
+        changes = len(out_lines) - 1 if len(out_lines) >= 1 else 0
```

Comment:
for better accuracy and consistency, this should be computed from the `changed` and `out` values, and the `changed_marker`, whose it's the purpose :) Maybe something as simple as
```suggestion
        changes = out.count(changed_marker) if changed else 0
```